examples/cardano/start_test.go

```diff
@@ -0,0 +1,94 @@
+package cardano
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/strangelove-ventures/interchaintest/v8"
+	ada "github.com/strangelove-ventures/interchaintest/v8/chain/cardano"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestCardano(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+
+	t.Parallel()
+
+	client, network := interchaintest.DockerSetup(t)
+	ctx := context.Background()
+
+	cf := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
+		{ChainConfig: ada.DefaultConfig("cardano")},
+	})
+
+	chains, err := cf.Chains(t.Name())
+	require.NoError(t, err)
+	adaChain := chains[0].(*ada.AdaChain)
+	err = adaChain.SetFaucet(ctx)
```

Comment:
UX nit: this (SetFaucet) likely makes more sense within the Build(...), similar to what we do for [`CreatePenumbraClient` if chain is of that type](https://github.com/strangelove-ventures/interchaintest/blob/137503860844eded95712df7cb26577047f772b1/interchain.go#L354).

Else this step would be fairly easy to miss since integrator may expect the faucet to already be setup. not a blocker